src/main/java/pl/common/todo/api/ToDoApi/model/Task.java

```diff
@@ -0,0 +1,60 @@
+package pl.common.todo.api.ToDoApi.model;
+
+public class Task {
+	
+	private Integer ID;
+	
+	private String name;
+	
+	private Integer deadline;
+	
+	private Integer dateOfInsertion;
+	
+	private Integer updateDate;
+
+	public Integer getID() {
+		return ID;
+	}
+
+	public void setID(Integer iD) {
+		ID = iD;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public Integer getDeadline() {
+		return deadline;
+	}
+
+	public void setDeadline(Integer deadline) {
+		this.deadline = deadline;
+	}
+
+	public Integer getDateOfInsertion() {
+		return dateOfInsertion;
+	}
+
+	public void setDateOfInsertion(Integer dateOfInsertion) {
+		this.dateOfInsertion = dateOfInsertion;
+	}
+
+	public Integer getUpdateDate() {
+		return updateDate;
+	}
+
+	public void setUpdateDate(Integer updateDate) {
+		this.updateDate = updateDate;
+	}
```

Comment:
Oprócz generowania getów/setów oraz toStringów, zawsze generujemy `hashCode` oraz `equals`. W pózniejszym czasie wyjaśnię Ci o co chodzi, bo na to jest za wcześnie. Prawdopodobnie.